app/models/mappers/check-activities.js

```diff
@@ -189,17 +196,30 @@ const getDateAndTeamMemberFromEvent = (event) => {
   }
 }
 
+/**
+ * @param {string|undefined|number|null} item
+ * @returns {string|number|null}
+ */
+const turnUndefinedOrEmptyToNull = (item) => {
+  if (item === undefined || item === '') {
+    return null
+  }
+  return item
+}
+
 /**
  * @param {AuditFieldRecord} auditFieldRecord
  * @returns {boolean}
  */
-const filterSameDate = (auditFieldRecord) => {
+const filterNonUpdatedFields = (auditFieldRecord) => {
   const [, before, after] = auditFieldRecord
 
-  if (before === after) {
+  // null, empty string and undefined should be the same
+  if (turnUndefinedOrEmptyToNull(before) === turnUndefinedOrEmptyToNull(after)) {
```

Comment:
Just wondering if null === null returns true using triple equals? I presume your unit tests covers that